pkg/configuration/configuration.go

```diff
@@ -187,6 +190,7 @@ func (c *Config) setConfigDefaults() {
 	c.host.SetDefault(varNotificationDeliveryService, NotificationDeliveryServiceMailgun)
 	c.host.SetDefault(varDurationBeforeNotificationDeletion, "24h")
 	c.host.SetDefault(varRegistrationServiceURL, "https://registration.crt-placeholder.com")
+	c.host.SetDefault(varEnvironment, "dev")
```

Comment:
We have to use `prod` by default or add `prod` to `deploy/env/prod.yaml`. I think it's less error prone to use `prod` here:
```suggestion
	c.host.SetDefault(varEnvironment, "prod")
```

Reply:
ohhh okay makes sense, i see that the registration service does the same. changed here: 9e8605c8ad04454dac98359c92913e9abdedc64d